Misc/NEWS.d/next/Core and Builtins/2024-06-12-13-47-25.gh-issue-120397.n-I_cc.rst

```diff
@@ -0,0 +1,2 @@
+Improve the :meth:`str.count`, :meth:`bytes.count` and :meth:`bytearray.count`
```

Comment:
"Improve"? How? Please elaborate. I suppose that you mean "Improve the speed".

Reply:
Ah that's my bad for this one. I suggested this wording but did not consider the "improve the speed" since I assumed that it would be put in the "optimizations" section of the whats new but it's not the case.

Reply:
How much? Does it make it 10% faster, 10 times faster?

Reply:
Up to 2x I would say. At least on my x86-64 machine.